pkg/okteto/client.go

```diff
@@ -96,11 +102,26 @@ func NewOktetoClientFromUrl(url string) (*OktetoClient, error) {
 		return nil, err
 	}
 
-	httpClient := oauth2.NewClient(context.Background(), nil)
+	ctx := backgroundContextWithHttpClient(config.IsInsecureSkipVerify())
+	httpClient := oauth2.NewClient(ctx, nil)
 
 	return newOktetoClientFromGraphqlClient(u, httpClient)
 }
 
+func backgroundContextWithHttpClient(insecureSkipVerify bool) context.Context {
```

Comment:
I would add a comment here as to why we are setting the client in the context as per [the docs](https://cs.opensource.google/go/x/oauth2/+/0ebed06d:oauth2.go;l=332-341). It wasn't immediately obvious without drilling into the oauth2 package code. Specially since if we add a token source at some point it won't be used